### ISSUE TITLE
Ansible Lint never runs / runs against wrong roles

### ISSUE THREAD

**Issue by techraf** [metacloud/molecule#512]
Situation
----
I have:

- "development" roles in: `~/molecule/`
- "production" roles in: `/usr/local/etc/ansible/roles/` (synchronised with Git)

My global `ansible.cfg` points to `/usr/local/etc/ansible/roles/` and Molecule (by design) maintains its own `.molecule/ansible.cfg` with: `roles_path      = .molecule/roles:.molecule/../roles:../:../../`.

Problem
----
During the verification phase, Molecule executes:
```
/usr/local/bin/ansible-lint playbook.yml
```
`ansible-lint` has no clue about `.molecule/ansible.cfg` and checks `my-role` in the globally defined path `/usr/local/etc/ansible/roles/`, not the one in the Molecule project directory (i.e. not from the `./../`.

Moreover, if `ansible-lint` can't find the role specified in the `playbook.yml` it gracefully exits with return code `0`.

Workaround
----
<strike>Create an additional `ansible.cfg` in the Molecule project directory (not under `.molecule` subdirectory) with `roles_path` pointing to `..`</strike>

Setting the value of `ANSIBLE_CONFIG` temporarily when calling `molecule`. Namely creating an alias:

```
alias molecule="ANSIBLE_CONFIG=.molecule/ansible.cfg molecule"
```

Possible fixes
----
- <strike>Call `/usr/local/bin/ansible-lint` after `chdir` to `.molecule` and call it against `../playbook.yml` (?)</strike>
- Add the `-r` option to Ansible Lint call ...except that... it does not seem to work (!)
- Call `ansible-lint` setting the value of `ANSIBLE_CONFIG` first:

    ```
    ANSIBLE_CONFIG=.molecule/ansible.cfg /usr/local/bin/ansible-lint playbook.yml
    ```

Other remarks
----
The only situation in which Molecule and Ansible Lint work correctly together is when the global `ansible.cfg` contains a reference to the directory containing Molecule projects (which might be true for many, but was not the case for me with the "development" and "production" split).

**Comment by retr0h** [comment]
> ANSIBLE_CONFIG=.molecule/ansible.cfg /usr/local/bin/ansible-lint playbook.yml

I'll update molecule to handle this use case.

**Comment by retr0h** [comment]
@techraf would you be able to test [this PR](https://github.com/metacloud/molecule/pull/514), and verify it works for your case?

**Comment by retr0h** [comment]
@techraf I released 1.11.5 to pypi with this functionality.
